wd_core.au3

```diff
@@ -811,13 +811,13 @@ EndFunc   ;==>_WD_ElementAction
 ; #FUNCTION# ====================================================================================================================
 ; Name ..........: _WD_ExecuteScript
 ; Description ...: Execute Javascipt commands.
-; Syntax ........: _WD_ExecuteScript($sSession, $sScript[, $sArguments = Default[, $bAsync = Default[, $bValueNode = False]]])
+; Syntax ........: _WD_ExecuteScript($sSession, $sScript[, $sArguments = Default[, $bAsync = Default[, $sJSONNode = Default]]])
 ; Parameters ....: $sSession   - Session ID from _WD_CreateSession
 ;                  $sScript    - Javascript command(s) to run
 ;                  $sArguments - [optional] String of arguments in JSON format
 ;                  $bAsync     - [optional] Perform request asyncronously? Default is False
-;                  $bValueNode - [optional] Return value node from Webdriver response. Default is False.
-; Return values .: Success - Raw response from web driver or value node.
+;                  $sJSONNode  - [optional] Return value according to given $sJSONNode instead raw response. Default is ""

```

Comment:
This doesn't read well to me. Maybe change to something like the following instead --

> Return the designated JSON node instead of the entire JSON string

Reply:
done